esphome/components/mcp4728/mcp4728.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "esphome/components/i2c/i2c.h"
+#include "esphome/components/output/float_output.h"
+#include "esphome/core/component.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace mcp4728 {
+
+extern const uint8_t MCP4728_MULTI_IR_CMD;
+extern const uint8_t MCP4728_MULTI_EEPROM_CMD;
+extern const uint8_t MCP4728_FAST_WRITE_CMD;
+
+typedef enum pd_mode {
+  MCP4728_PD_MODE_NORMAL,
+  MCP4728_PD_MODE_GND_1K,
+  MCP4728_PD_MODE_GND_100K,
+  MCP4728_PD_MODE_GND_500K
+} MCP4728_pd_mode_t;
```

Comment:
```suggestion
enum MCP4728PDMode {
  MCP4728_PD_MODE_NORMAL,
  MCP4728_PD_MODE_GND_1K,
  MCP4728_PD_MODE_GND_100K,
  MCP4728_PD_MODE_GND_500K
};
```

Here we're in C++ and we don't need to use typedef enums :)

Reply:
Done